web/templates/personal_menu.hbs

```diff
@@ -0,0 +1,135 @@
+<div class="dropdown-menu" id="personal-menu-dropdown" data-simplebar>
+    <nav class="inner">
+        <header class="user-info-popover-header">
+            <div class="avatar">
+                <img class="avatar-image{{#if user_is_guest}} guest-avatar{{/if}}" src="{{user_avatar}}"/>
+
+                {{#if is_active }}
+                <span class="status-circle {{user_circle_class}} user_circle hidden-for-spectators" data-tippy-placement="bottom" data-tippy-content="{{user_last_seen_time_status}}"></span>
+                {{/if}}
+
+            </div>
+            <div class="text-area">
+                <p class="full-name">{{user_full_name}}</p>
+                <p class="user-type">{{user_type}}</p>
+            </div>
+        </header>
+        <section class="popover-menu personal-menu-actions" data-user-id="{{user_id}}">
+            <ul class="list">
+                <li class="text-item hidden-for-spectators">
+                    <i class="zulip-icon zulip-icon-clock"></i>
+                    {{#tr}}{user_time} local time{{/tr}}
+                </li>
+            </ul>
+
+            <ul class="list">
+                {{#if status_content_available}}
+                <li class="text-item">
+                    <span>
+                        {{#if status_emoji_info}}
+                            {{#if status_emoji_info.emoji_alt_code}}
+                            <span class="emoji_alt_code">&nbsp;:{{status_emoji_info.emoji_name}}:</span>
+                            {{else if status_emoji_info.url}}
+                            <img src="{{status_emoji_info.url}}" class="emoji status_emoji" />
+                            {{else}}
+                            <span class="emoji status_emoji emoji-{{status_emoji_info.emoji_code}}"></span>
+                            {{/if}}
+                        {{/if}}
+                        <span class="status_text">
+                            {{status_text}}
+                        </span>
+                    </span>
+                    <a href="" class="clear_status" aria-label="{{t 'Clear status'}}" data-tippy-content="{{t 'Clear your status' }}">
+                        <i class="zulip-icon zulip-icon-x-circle"></i>
+                    </a>
+                </li>
+                <li class="link-item">
+                    <a href="" class="update_status_text">
+                        <span class="link-item-focus">
```

Comment:
Can you explain why we need this `link-item-focus` class to be a separate `span`? If we're going to be using these styles in all of our popovers eventually, it feels like a lot of potential.

Probably a thing to discuss in a thread in #frontend in case there's a way to avoid whatever constraint made you want to do this.

Reply:
https://chat.zulip.org/#narrow/stream/6-frontend/topic/personal.20menus.20focus.20emulation